packages/core/src/codewhisperer/tracker/lineTracker.ts

```diff
@@ -48,16 +48,18 @@ export class LineTracker implements vscode.Disposable {
 
     constructor() {
         this._disposable = vscode.Disposable.from(
-            vscode.window.onDidChangeActiveTextEditor(e => {
-                this.onActiveTextEditorChanged(undefined)
+            vscode.window.onDidChangeActiveTextEditor(async e => {
+                await this.onActiveTextEditorChanged(undefined)
             }),
```

Comment:
for future reference: return the promise should also work
```suggestion
            vscode.window.onDidChangeActiveTextEditor(e => {
                return this.onActiveTextEditorChanged(undefined)
            }),
```

Reply:
ack, knowledge learned. 
Are these 2 equivalent?

Reply:
yes. but `async` is useful if you are calling mutliple functions.